dataprofiler/data_readers/csv_data.py

```diff
@@ -106,6 +106,10 @@ def quotechar(self):
     def header(self):
         return self._header
 
+    @property
+    def is_structured(self):
+        return self.data_format == "dataframe"
```

Comment:
I think this should just always be true? Isnt record format also structured?

Reply:
^ Yes

Reply:
@JGSweets can you confirm? I thought I remember you saying it was dependent on data format and that records were a way to strip away structure?

Reply:
@ChrisWallace2020 is correct. Records is a list of strings, basically treating it as a text file.

Reply:
The intent is for the profiler to know how to profile the dataset. Records would not be able to be profiled by the structured profiler